docs/content/users/quickstart.md

```diff
@@ -375,6 +375,31 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
     For more advanced tasks like adding elasticsearch, building and watching storefront and administration, see [susi.dev](https://susi.dev/ddev-shopware-6).
 
+=== "Silverstripe"
+
+    ## Silverstripe
+
+    === "Silverstripe"
+
+        Use a new or existing Composer project, or clone a Git repository.
+
+        === "Composer"
+            ```bash
+            mkdir my-silverstripe-app
+            cd my-silverstripe-app
+            ddev config --project-type=silverstripe --docroot=public --create-docroot
+            ddev composer create --prefer-dist --no-scripts silverstripe/installer -y
+            ddev start
```

Comment:
I've downloaded the current artifact of the PR and manually tested the step. Beofre the `ddev launch /admin` another step is missing and should be added to the quickstarted guides:
`ddev exec vendor/bin/sake dev/build`

without the cmd the user runs into an error when trying to launch the site because the database isn't populated yet.

Reply:
Just a note that `vendor/bin` is in the web container $PATH already, so `ddev exec sake` should work.

Reply:
I've quickly tested and it works to just use `ddev exec sake dev/build`. it is tempting to shorten the command. but on second thought i think it might be a good idea to keep the relational path. that way it is easier reproducible for someone to understand where the sake command resides within the project directory inside the container. for me `ddev exec vendor/bin/sake dev/build` is already cryptic for example but having `ddev exec sake dev/build` instead feels even more like voodoo imho.

Reply:
I like people to know that $PATH works and that vendor/bin is in the path. That's how drush works, for example. You certainly don't have to `ddev exec vendor/bin/drush` - true of many of the other quickstarts here.

Reply:
And $PATH isn't voodoo :)

Reply:
but the difference with drush is that is something that you install in the quickstarter while sake ships with silverstripe. but basically you are right. maybe the better approach might be using the short form and explain in the liner notes for the quickstart guide where for example is located within the project. 

and with voodoo i just meant when you read the instructions and see something unusual like sake or /vendor/bin/sake the latter makes it a bit more graspable. ;)

Reply:
Note that a `sake` shorthand is included in this PR.

I've added some of your comments around it to the quickstart.